### ISSUE TITLE
Android app startup performance slow

### ISSUE THREAD

**Issue by sreich** [bitwarden/mobile#42]
Android startup performance is much slower than lastpass, which I thought was slow in itself.

A cold start is much slower than a cold start of lastpass, and a hot start of lastpass is about 2 seconds.

A hot start of bitwarden is 3-6 seconds. Obviously this isn't a benchmark and you and I know the many variables in the system, but I can feel it's significantly slower and too slow when I have to rely on it for logging into everything.

This is on a pixel XL running Android 7.1.2(beta)

This was also with a near empty database, without any locking enabled. Lastpass I think is faster if I didn't have fingerprint enabled.

I'm free to test or debug (like provide logs, etc) if necessary. I can't speak as to if this is a regression or not (I haven't used it significantly since I was waiting on auto fill)

**Comment by kspearrin** [comment]
What are you considering as a hot start? It seems mostly instant to me if i background the app and re-open it.

**Comment by kspearrin** [comment]
@jerryn70 What are your device details and versions?

**Comment by jerryn70** [comment]
Moto x play 
Marshmallow 6.0.1
Non rooted

**Comment by kspearrin** [comment]
@jerryn70 I've seen similar behavior once a while back and after I reinstalled the app it went away. I haven't seen it since though. I'll continue to investigate.

**Comment by sreich** [comment]
I was referring to hot start as in me being in the app and hitting home, or back to get back to the launcher.

As opposed to swiping the app out of recents which kills the app.

The latter is where the lowest hanging fruit are. Its startup after it is swiped from recents(what I would call a cold start) is quite long (esp when I'm waiting to log into my computer each bathroom break)

I've also noticed it's using about 377 MB of RAM right now..That probably doesn't help things as far as getting memory killed goes.

See http://i.imgur.com/P1PJ4Br.png

I don't know how much of it is in your control or inherent with xamarin. 

Although I did find it interesting that this page shows two bitwarden entries.. Which is strange to me, maybe one is the accessibility service(LastPass doesn't have that though)? Or perhaps separate threads?

**Comment by kspearrin** [comment]
This should now be resolved with the 491 build in the play store beta.

**Comment by i3130002** [comment]
@sreich Is it a duplicate of https://github.com/bitwarden/mobile/issues/1723 ?

**Comment by sreich** [comment]
I wouldn't say so, theirs is more about biometric unlock, this issue was about startup. But, either way this seems to be resolved I don't have any issues with it on my pixel 6a
